tmail-backend/tmail-third-party/openpaas/pom.xml

```diff
@@ -67,6 +72,11 @@
             <artifactId>mockserver-netty</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.linagora.tmail</groupId>
+            <artifactId>tmail-guice-distributed</artifactId>
```

Comment:
That seems to create a cyclic dependency issue, as reported by the CI build =>

[ERROR] [ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='com.linagora.tmail:tmail-openpaas:1.0.0-SNAPSHOT'}' and 'Vertex{label='com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT'}' introduces to cycle in the graph com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT --> com.linagora.tmail:tmail-openpaas:1.0.0-SNAPSHOT --> com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT @ 
[ERROR] The projects in the reactor contain a cyclic reference: Edge between 'Vertex{label='com.linagora.tmail:tmail-openpaas:1.0.0-SNAPSHOT'}' and 'Vertex{label='com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT'}' introduces to cycle in the graph com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT --> com.linagora.tmail:tmail-openpaas:1.0.0-SNAPSHOT --> com.linagora.tmail:tmail-guice-distributed:1.0.0-SNAPSHOT -> [Help 1]